baking/Cornmeal_Molasses_Scones.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "Cornmeal Molasses Scones"
+source_url: "https://www.seductioninthekitchen.com/cornmeal-molasses-scones/"
+oven_temp: "370 F"
+time: "60m"
+yield: "9 inch pie pan"
+---
+
+# Ingredients
+
```

Comment:
Would you mind tighting up this whitespace? I prefer one space after a title and two spaces before a title. I should set up `mdlint` (or some equivalent in this repo)